docs/adr/components-for-policy-scanning.md

```diff
@@ -0,0 +1,55 @@
+# Component Versioning for Openstack Entities
+
+## Context and Problem Statement
+
+How to accomodate Openstack entities like Security Groups in the existing Entity Relationship of Heureka? 
+
+The current Entity Relationships in Heureka are defined for vulnerabilities in objects like containers, where it is intuitive to have ComponentVersion and ComponentInstances. For non-compliance (not exactly vulnerabilities) in entities like Security Group, how to adapt to existing data model?
+
+## Decision Drivers
+
+* Fits into existing data model 
+* Make use of Heureka Issue Matching through Component Version
+
```

Comment:
I'd suggest: 

* Avoid specialized data models 
* Enables addition of OpenStack Resources
* Allows Re-Use of existing business logic

Reply:
ADR Revised